dlt-database/entity/0001-init_db/Account.ts

```diff
@@ -8,7 +8,7 @@ import {
   BaseEntity,
 } from 'typeorm'
 import { User } from '../User'
-import { TransactionRecipe } from '../TransactionRecipe'
+import { TransactionRecipe } from './TransactionRecipe'
```

Comment:
always refere on the entity-files in the directory `/dlt-database/entity` instead out of the versioned directories, because new version of the entity `TransactionRecipe` without changings  in the entity `Account` will not use the new version of `TransactionRecipe`

you could define a path `@entity` to the directory `/dlt-database/entity` and use this path notation instead.

Reply:
That is by design, because actually TransactionRecipe was deleted. 
Or rather merged with ConfirmedTransaction to one table simply named Transaction. 
Working with roles I noticed that TransactionRecipe and ConfirmedTransaction are actually roles. So merging it into one Table make my code a lot easier.

Reply:
then please add a comment above the code, because in the near future it will be hard to follow why you make this change...

Reply:
done.